resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala

```diff
@@ -99,6 +99,26 @@ private[mesos] object MesosSchedulerBackendUtil extends Logging {
     .toList
   }
 
+  /**
+   * Parse a list of docker parameters, each of which
+   * takes the form key=value
+   */
+  private def parseParamsSpec(params: String): List[Parameter] = {
+    params.split(",").map(_.split("=")).flatMap { spec: Array[String] =>
```

Comment:
to be consistent with other methods, i think we should stick with spec.

>   def parsePortMappingsSpec(portmaps: String): List[DockerInfo.PortMapping] = {
>     portmaps.split(",").map(_.split(":")).flatMap { spec: Array[String] =>
>       val portmap: DockerInfo.PortMapping.Builder = DockerInfo.PortMapping

it seems to be better to reserve the name parameter later for builder

> val param: Parameter.Builder = Parameter.newBuilder()